integration-test/src/main/java/com/sequenceiq/it/cloudbreak/SshUtil.java

```diff
@@ -0,0 +1,99 @@
+package com.sequenceiq.it.cloudbreak;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import net.schmizz.sshj.SSHClient;
+import net.schmizz.sshj.common.IOUtils;
+import net.schmizz.sshj.connection.channel.direct.Session;
+import net.schmizz.sshj.transport.verification.PromiscuousVerifier;
+
+
+public class SshUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(com.sequenceiq.it.cloudbreak.SshUtil.class);
+
+    public Boolean ssh(String host, String defaultPrivateKeyFile, String sshCommand, String checkType, String value) {
+        SSHClient sshClient = null;
+        boolean result = false;
+        try {
+            sshClient = createSSHClient(host, 22, "cloudbreak", defaultPrivateKeyFile);
+            Pair<Integer, String> cmdOut = executeSshCommand(sshClient, sshCommand);
+            LOGGER.info("Ssh command status code and output: " + cmdOut.toString());
+            result = cmdOut.getLeft() == 0 && checkCommandOutput(cmdOut, checkType, value);
+        } catch (Exception ex) {
```

Comment:
Should be better to catch only IOException (the checked exception) and not take care of runtime exceptions, test framework should handle them